modules/transcription-service-workflowoperation/src/main/java/org/opencastproject/transcription/workflowoperation/NibityAttachTranscriptionOperationHandler.java

```diff
@@ -175,4 +172,35 @@ public void setWorkspace(Workspace service) {
     this.workspace = service;
   }
 
+  public MediaPackage addTranscriptToMediaPackage(InputStream zis, String captionMimeType, String captionIdentifier, String captionFileType, MediaPackage mediaPackage, MediaPackageElementFlavor flavor, String targetTagOption) throws WorkflowOperationException {
```

Comment:
This is used both to add a transcript and captions so it would be clearer to give it a more generic name, e.g. addElementToMediaPackage or addTranscriptionElementToMediaPackage

Reply:
Also avoid super-long lines.